elements/rh-video-embed/docs/10-style.md

```diff
@@ -22,7 +22,7 @@ The video embed element consists of a semitransparent play button with a video t
 
 ## Theme
 
-The video embed element is available in both light and dark themes. It can include either a light or a dark play button, depending on the thumbnail image underneath. If an image is lighter, use a dark play button for accessibility and vice versa.
+The video embed element is available in both light and dark themes. It can include either a light or a dark play button, depending on the thumbnail image underneath. If an image is lighter, use a dark play button for accessibility and vice versa. The user is currently responsible for setting the play button color and preventing color contrast issues between the button and the video thumbnail.
```

Comment:
```suggestion
The video embed element is available in both light and dark themes. It can include either a light or a dark play button, depending on the thumbnail image underneath. If an image is lighter, use a dark play button for accessibility and vice versa. The user is currently responsible for setting the play button color and preventing color contrast issues between the button and the video thumbnail. For example, if the video embed uses a dark color palette or is on a dark background, the play button colour must be light, and vice-versa.
```